### PR TITLE
Increase mem limit of single-HCD set up from 2 to 3 gigs; DSEx3 to 2.5 gigs per node

### PR DESCRIPTION
**Pull request by tatu-at-datastax**
**What this PR does**:

Increases max mem for HCD-backed single node cluster

**Which issue(s) this PR fixes**:
N/A

**Checklist**
- [x] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Comment by jeffreyscarpenter** [comment]
Should we also update the docker-compose/README to state minimum resources required for Docker Desktop to run these configurations?

**Comment by tatu-at-datastax** [comment]
> Should we also update the docker-compose/README to state minimum resources required for Docker Desktop to run these configurations?

@jeffreyscarpenter added notes -- WDYT?

File: docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ services:
     image: datastax/dse-server:${DSETAG}
```
Comment (jeffreyscarpenter) [inline docker-compose/docker-compose.yml:5]: can we add 

```
    platform: linux/amd64
```

File: docker-compose/docker-compose-hcd.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ services:
     image: datastax/hcd:${HCDTAG}
```
Comment (jeffreyscarpenter) [inline docker-compose/docker-compose-hcd.yml:5]: Can we add

```
    platform: linux/amd64
```

**Review by jeffreyscarpenter — APPROVED** [review]

**Pull request merged**